recipes/flake8-sfs/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "flake8-sfs" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "aeada40b526bf49d443f3f21483705dd8fa958c23809671e2ea8ab8cc1eeb4ab" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
`fn` is not required/used anymore with the newer `conda-builda` versions, so please remove it.